UPGRADING.md

```diff
@@ -0,0 +1,6 @@
+# UPGRADING
+
+## [UPGRADING 2.x to 3.0]
+### SearchAfter of SearchRequest type
+- Changed SearchAfter of SearchRequest type to FieldValue instead of String
+- Consider using `FieldValue.of("")` to make string type values compatible.
```

Comment:
Since this is the first time we're doing this, can we try to be extra helpful to the users?

I think we need something like https://github.com/ruby-grape/grape/blob/master/UPGRADING.md#recognizing-path, which has 1) Text that explains the change, a link to the PR (this PR), a before and after.